### PR TITLE
Move actor which removes "leapp-resume" service into After phase

### PR DESCRIPTION
**Pull request by Rezney**
In order to avoid killing (by removal) the "leapp-resume" service
and at the same time trying to reload systemd daemon by other
services, we move the actor which removes "leapp-resume" service
into FirstBootPhase.After phase.

https://bugzilla.redhat.com/show_bug.cgi?id=1883218

### CODE REVIEW

**Comment by pirat89** [comment]
Btw, if we do that, we should be sure that nobody else will use this specific After phase, otherwise there will be still the problem. We can consider it as a workaround, but in future I suggest to create new phase, which we document it's just for us, or we will specify virtual "endphase" as analogy to our "configuration" phase. But I am ok to merge it as a temporary workaround if it's planned for the upcoming release. In such a case, I recommend to create a task for that before the merge @Rezney

**Comment by Rezney** [comment]
The workaround seems to be working fine.

```
Oct 19 17:05:43 leapp-20201019174001 leapp3[5542]: 2020-10-19 17:05:43.621 INFO     PID: 5542 leapp.workflow.FirstBoot: Starting stage After of phase FirstBoot
Oct 19 17:05:43 leapp-20201019174001 leapp3[5542]: 2020-10-19 17:05:43.669 INFO     PID: 5542 leapp.workflow.FirstBoot: Executing actor remove_systemd_resume_service
Oct 19 17:05:43 leapp-20201019174001 leapp3[5542]: 2020-10-19 17:05:43.763 DEBUG    PID: 5996 leapp.workflow.FirstBoot.remove_systemd_resume_service: External command has started: ['systemctl', 'disable', 'leapp_resume.service']
Oct 19 17:05:43 leapp-20201019174001 leapp3[5542]: Removed /etc/systemd/system/default.target.wants/leapp_resume.service.
Oct 19 17:05:43 leapp-20201019174001 systemd[1]: Reloading.
Oct 19 17:05:44 leapp-20201019174001 leapp3[5542]: 2020-10-19 17:05:44.54  DEBUG    PID: 5996 leapp.workflow.FirstBoot.remove_systemd_resume_service: External command has finished: ['systemctl', 'disable', 'leapp_resume.service']
Oct 19 17:05:44 leapp-20201019174001 leapp3[5542]: 2020-10-19 17:05:44.323 INFO     PID: 5542 leapp: Answerfile will be created at /var/log/leapp/answerfile
Oct 19 17:05:44 leapp-20201019174001 leapp3[5542]: Debug output written to /var/log/leapp/leapp-upgrade.log
Oct 19 17:05:44 leapp-20201019174001 leapp3[5542]: ============================================================
Oct 19 17:05:44 leapp-20201019174001 leapp3[5542]:                            REPORT
Oct 19 17:05:44 leapp-20201019174001 leapp3[5542]: ============================================================
Oct 19 17:05:44 leapp-20201019174001 leapp3[5542]: A report has been generated at /var/log/leapp/leapp-report.json
Oct 19 17:05:44 leapp-20201019174001 leapp3[5542]: A report has been generated at /var/log/leapp/leapp-report.txt
Oct 19 17:05:44 leapp-20201019174001 leapp3[5542]: ============================================================
Oct 19 17:05:44 leapp-20201019174001 leapp3[5542]:                        END OF REPORT
Oct 19 17:05:44 leapp-20201019174001 leapp3[5542]: ============================================================
```

@shaded-enmity @mkluson ^^^

**Review by pirat89 — APPROVED** [review]
lgtm and it seems working

**Review by vinzenz — APPROVED** [review]

**Comment by AloisMahdal** [comment]
Verified manually by @mkluson .  From From QE POV -- good to merge; thanks!

**Pull request merged**
